client/src/components/AddNewProfile.js

```diff
@@ -16,46 +17,96 @@ const AddNewProfile = () => {
     support_type: "",
     profile_type: "",
     status: "new",
-    join_date: new Date()
+    nationality: "",
+    join_date: new Date(),
   });
```

Comment:
@NataliaCoder `address` and `occupation` can be empty so we don't need to do validation for them in the frontend. First, we need to make some changes so that the database is okay with empty values for them.

Right now, `recreate-schema.sql` has the following for the `profiles` table:
```
  address         VARCHAR(200) CHECK (address <> ''),
  phone_number    VARCHAR(50) CHECK (length(phone_number) > 9),
  occupation      VARCHAR(100) CHECK (occupation <> ''),
```

We need to remove the checks for `address` and `occupation` like:
```
  address         VARCHAR(200)
  phone_number    VARCHAR(50) CHECK (length(phone_number) > 9),
  occupation      VARCHAR(100)
```

Can you do these changes in this pull request? This way, the app is okay if the user sends an empty `address` and `occupation` on the Add New Profile page. Hope that makes sense!